godswillumukoro/browser-events-manager
# Add external replication server for development and testing

This PR introduces a simple Express server that simulates an external data replication service. It includes:

- A POST endpoint (/backup/replicate) for receiving and storing replicated data
- A GET endpoint (/backup/data) for retrieving all stored data
- Basic error handling and data validation
- Server configuration using environment variables
 
This server is intended for development and testing purposes to simulate the external API that the Browser Events Manager extension will interact with for data replication.
 
Testing Instructions:
1. Run `npm install express dotenv`
2. Start the server with `node server/app.js`
3. Use a tool like Postman to send POST requests to `http://localhost:4999/backup/replicate`
4. Verify stored data by sending a GET request to `http://localhost:4999/backup/data`
